Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:18-alpine as builder
+
+ARG BUILD_CONTEXT
+
+WORKDIR /base
+
+COPY package.json .
+COPY yarn.lock .
+COPY ./packages ./packages
+
+RUN yarn install
+
+RUN yarn build:$BUILD_CONTEXT
+
+FROM node:18-alpine as base-runner
```

Comment:
Probably not the best solution to copy all packages to build only one. We need to copy only a service and its local dependencies. Any idea how to do it?